src/_includes/scripts/index.js

```diff
@@ -4,11 +4,11 @@ var GA_ANALYTICS_ID = "G-70EPL8ZB7Y";
 
 function init() {
   var gaCookieValue = getCookie(COOKIE_NAME);
-  var isAnalyticsCookieSet = gaCookieValue !== null;
+  var isAnalyticsCookiesAccepted = gaCookieValue == "true";
 
-  setupClickEvent(isAnalyticsCookieSet);
+  setupClickEvent(isAnalyticsCookiesAccepted);
 
-  if (isAnalyticsCookieSet) {
+  if (gaCookieValue !== null) {
     toggleCookieBanner();
     if (gaCookieValue == "true") {
```

Comment:
you can replace this now with isAnalyticsCookiesAccepted

Reply:
Not in that place as that shows the cookie banner if it isn't set yet, however line 13 I can!